landable.gemspec

```diff
@@ -37,6 +37,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'builder'
   gem.add_dependency 'lookup_by', '> 0.4.0'
   gem.add_dependency 'highline'
+  gem.add_dependency 'hutch'
```

Comment:
I do not think we need Hutch as a dependency as hutch enabled is defaulting to false.  Can we just have hutch be included on the parent app and not landable itself?